packages/styles/components/_c.listbox.scss

```diff
@@ -0,0 +1,145 @@
+    width: $mu025;
+
+    &-thumb {
+      background: $color-grey-600;
+    }
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    padding-left: $mu075;
+    min-height: $mu300;
+    border-bottom: 1px solid $color-grey-300;
+
+    @include set-focus-floating-base() {
+      border-radius: 0;
+      top: $mu025;
+      right: $mu025;
+      bottom: $mu025;
+      left: $mu025;
+    }
+
+    &:hover {
+      background-color: $color-grey-100;
+      box-shadow: inset 9px 0 0 -7px $color-grey-900;
+      cursor: pointer;
+    }
+
+    &--checked {
+      pointer-events: none;
+      background-size: $mu150;
+      background-image: url(inline-icons(
+        "notification-available-24",
+        $color-grey-900
+      ));
+      background-repeat: no-repeat;
+      background-position: right $mu075 center;
+    }
+  }
+
+  &__text {
+    pointer-events: none;
+  }
+
+  &__icon {
+    pointer-events: none;
+    width: $mu200;
+    height: $mu200;
+    margin-right: $mu075;
+  }
+
+  &__empty {
+    align-items: center;
+    border: 1px solid $color-grey-600;
+    border-radius: get-border-radius("m");
+    padding: $mu100 $mu050;
+  }
+
+  .is-focus {
+    @include set-focus-floating();
+  }
+
+  .is-disabled {
+    cursor: not-allowed;
+    box-shadow: none;
+    background-color: $color-grey-200;
+    color: $color-grey-600;
+  }
+
+  &--multi {
+    #{$parent} {
+      &__item {
+        position: relative;
+      }
+
+      &__input {
+        border-color: $color-input-border;
+        position: absolute;
+        right: $mu100;
+        top: $mu100;
+
+        @include reset-input;
+
+        min-width: $checkbox-min-width;
+        min-height: $checkbox-min-height;
+        width: $checkbox-width;
+        height: $checkbox-width;
+        border-radius: $checkbox-border-radius;
+        border: $checkbox-border-width solid $color-input-border;
+        background: $color-input-background;
+
+        &:checked {
+          background-color: $color-input-checked-background;
+          border-color: $color-input-checked-border;
+          background-image: url(inline-icons(
+            "checkmark-16",
+            $color-input-checked-icon
+          ));
+          background-position: center center;
+          background-size: $mu100;
+
+          &::-ms-check {
+            background-color: $color-input-checked-background;
+            border-color: $color-input-checked-border;
+          }
+
+          &:disabled {
+            border-color: $color-input-disabled-border;
+            background: $color-input-disabled-background;
+            background-image: url(inline-icons(
+              "checkmark-16",
+              $color-input-disabled-icon
+            ));
+          }
+        }
+      }
+
+      &__label {
+        cursor: pointer;
+        align-self: stretch;
+        display: inline-flex;
+        align-items: center;
+        width: 100%;
+      }
```

Comment:
From my point of view, the `&__text` and `&__label` elements should be "merged" and use a single class. Only the tag should change depending on the context.